libvast/src/system/type_registry.cpp

```diff
@@ -49,8 +49,11 @@ type_registry_state::status(status_verbosity v) const {
     caf::put(tr_status, "types", keys);
     // The list of defined concepts
     auto& concepts_status = put_dictionary(tr_status, "concepts");
-    for (auto& concept_ : taxonomies.concepts)
-      concepts_status[concept_.first] = concept_.second;
+    for (auto& concept_ : taxonomies.concepts.data) {
```

Comment:
```suggestion
    for (auto& [x, y] : taxonomies.concepts.data) {
```

Please do this with appropriate variable names. Kind of hard to figure out what first and second are below.